03/app.js

```diff
@@ -0,0 +1,21 @@
+const btn = document.querySelectorAll('button');
+console.log(btn)
+
+
+const clickBtn = function () {
+    this.innerText = 'clicked';
+    console.log('clicked')
+
+    btn.forEach(function (btnItem) {
+        btnItem.removeEventListener('click', clickBtn)
+
+    })
+}
+
```

Comment:
powinno to wyglądać tak:
```
const clickBtn = function () {
    this.innerText = 'clicked';
    this.removeEventListener('click', clickBtn)
    console.log('clicked')
}
```
Czyli tylko z tego klikniętego usuwamy nasłuchiwanie